news/templates/news/event.html

```diff
@@ -75,7 +81,12 @@
             {% if article.standalone and article.timeplace_set.future %}
                 <div class="ui card">
                     {% if article.number_of_tickets %}
-                        {% if article.number_of_registered_tickets < article.number_of_tickets %}
+                        {% is_registered article user as registered %}
+                        {% if registered %}
+                            <a href="{% url "my-tickets" %}" class="ui green fluid button">
```

Comment:
Wouldn't it be better to link to the page for that specific ticket?

Reply:
Yes, then it will be the same as the redirect after registering.